decidim-core/app/presenters/decidim/metric_charts_presenter.rb

```diff
@@ -40,19 +40,32 @@ def render_not_highlighted(not_highlighted_metrics)
 
     def render_metrics_data(metric_name, opts = {})
       content_tag :div, class: opts[:klass].presence || "column medium-6" do
-        content_tag :div,
-                    "",
-                    id: "#{metric_name}_chart",
-                    class: "areachart metric-chart #{opts[:graph_klass]}",
-                    data: {
-                      chart: "areachart",
-                      metric: metric_name,
-                      info: {
-                        title: I18n.t("decidim.metrics.#{metric_name}.title"),
-                        object: I18n.t("decidim.metrics.#{metric_name}.object")
-                      }
-                    }
+        render_join_description(opts) if opts[:axis]
+        render_metric_chart(metric_name, opts)
       end
     end
+
+    def render_join_description(opts)
+      concat content_tag(:h3, opts[:title], class: "heading3 text-uppercase text-muted")
+      concat content_tag(:p, opts[:description], class: "text-medium")
+    end
+
+    def render_metric_chart(metric_name, opts = {})
+      concat content_tag :div,
+                         "",
+                         id: "#{metric_name}_chart",
+                         class: "areachart metric-chart #{opts[:graph_klass]}",
+                         style: opts[:margin],
+                         data: {
+                           chart: "areachart",
+                           metric: metric_name,
+                           ratio: (opts[:ratio]).to_s,
+                           axis: opts[:axis],
```

Comment:
Forcing data attributes is not recomended. Use default ones and the `merge` them.